src/utils/manifest.ts

```diff
@@ -42,13 +42,3 @@ export const getEntryPointFromManifest = async () => {
     ? result.packageJson.skuba.entryPoint
     : DEFAULT_ENTRY_POINT;
 };
-
-export const isBabelFromManifest = async () => {
```

Comment:
Is it worth keeping this around so we can emit a warning? I’m not sure if anyone ended up using Babel support.

Reply:
I'm comfortable with a clean break here; I've fixed up the one instance of internal SEEK usage and this has always been marked as "experimental".